packages/perseus-core/src/data-schema.ts

```diff
@@ -1343,6 +1343,9 @@ export type PerseusRadioWidgetOptions = {
     // If multipleSelect is enabled, Specify the number expected to be correct.
     // NOTE: perseus_data.go says this is required even though it isn't necessary.
     countChoices?: boolean;
+    // How many of the choices are correct, which is conditionally used to tell
+    // learners ahead of time how many options they'll need.
+    numCorrect?: number;
```

Comment:
Does updating this file require a stakeholder update? Apologies for pointing this out each time, but the memories of what Ned went through and Jeremy always talking about protecting this file are strong 😅

Reply:
Thanks for bringing this up: https://khanacademy.slack.com/archives/C49296Q7P/p1740496295273589